pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
+requires = ["setuptools", "wheel", "oldest-supported-numpy", "Cython"]
```

Comment:
I assume `oldest-supported-numpy` is a new way to avoid having to do
```
    "numpy==1.13.3; python_version=='3.5'",
    "numpy==1.13.3; python_version=='3.6'",
    "numpy==1.14.5; python_version=='3.7'",
    "numpy==1.17.3; python_version>='3.8'",
    ...
```
Can you point me to docs for this?

Reply:
Yes, that's the idea. It's here https://github.com/scipy/oldest-supported-numpy

I remember reading a thread about numpy pinnings during build time in the pypa list (or perhaps in scipy devel list), but I can't find it right now